src/components/ui/UserPicture.vue

```diff
@@ -77,15 +76,27 @@ export default {
         this.src = await generateIdenticon(this.avatar.username, this.size);
       } else {
         this.src = `${this.avatar.picture}?size=${this.slot}`;
+        this.srcset = this.buildSrcset();
       }
     },
+    buildSrcset() {
+      return [40, 100, 264]
+        .map(
+          (s) =>
+            `${this.avatar.picture}?size=${s} ${(s / this.slot).toPrecision(
```

Comment:
Since you can pass any number to ```this.slot```, we should probably do some checks before we divide by 0.

Reply:
Good point! I'll get that changed